src/Hook.php

```diff
@@ -22,10 +22,19 @@ public static function getValidHooks()
         );
         $validHooks = [];
 
+        //add customize hooks
+        $customizeHooks = isset($json['extra']['customize_hooks']) ?
```

Comment:
Okay, while that's great, it's not technically a git hook. It's a script that you're aliasing to a git command. So, it's not really in the scope of this project. Don't you think?